autorest/code_generator.py

```diff
@@ -69,9 +69,9 @@ def _build_exceptions_set(self, yaml_data):
                     exceptions_set.add(exception['schema']['language']['python']['name'])
         return exceptions_set
 
-    def _create_code_model(self, yaml_code_model):
+    def _create_code_model(self, yaml_code_model, azure_arm):
```

Comment:
I'm afraid if we pass more options later, it becomes impossible.
What about getting all options from autorest in a Dict[Enum, value] where we can do:
> options[AzureArm] = self._autorestapi.get_value("azure-arm")
and pass this options dict everywhere?

Reply:
makes sense, thanks!